documentation_builder/simulating.ipynb

```diff
@@ -758,13 +759,14 @@
       "PRODUCING REACTIONS -- Pyruvate\n",
       "-------------------------------\n",
       "  %             FLUX   RXN ID                        REACTION                       \n",
-      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
-      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
+      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
```

Comment:
Ok :) I would vote for  the min - max format since any center value could be interpreted as the mean/median/most likely flux, all of which are false. Also it could give the impression that taking the solution where each flux takes its center value yields the reported biomass objective, which is also not true...